SConstruct

```diff
@@ -1021,43 +1021,47 @@ env['python_cmd_esc'] = quoted(env['python_cmd'])
 
 # Python 2 Package Settings
 cython_min_version = LooseVersion('0.23')
+numpy_min_version = LooseVersion('1.6.1')
```

Comment:
According to `compiling.rst`, we expect Cantera to work with 1.4 or newer, although I don't know the last time it's been tried with anything older than 1.6. I'm just a bit leery of setting our version requirements based on what is installed on Travis (also note that we have the Buildbot builders as well at http://gir.mit.edu:8010/waterfall, although I think those are all running more recent versions of numpy).